core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -175,6 +175,14 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val EVENT_LOG_OLD_STACK_TRACE =
+    ConfigBuilder("spark.eventLog.logOldStackTrace")
+      .doc("Whether to write old format of stacktrace to the event log. " +
+        "Only enable this when spark history server is before version 1.2.0")
```

Comment:
Which version is being referred to here with `1.2.0` ?

Reply:
It's in `TaskEndReason`. The code is as below:
```
  override def toErrorString: String =
    if (fullStackTrace == null) {
      // fullStackTrace is added in 1.2.0
      // If fullStackTrace is null, use the old error string for backward compatibility
      exceptionString(className, description, stackTrace)
    } else {
      fullStackTrace
    }
```